### ISSUE TITLE
add to F-droid

### ISSUE THREAD

**Issue by cyanwolfg** [hardcore-sushi/AIRA-android#5]
I'm not sure if you want to still with development of this app but i think that if the app is in F-droid many people can see it a get help or ideas. I know and start to use DroidFS for that reason.

**Comment by hardcore-sushi** [comment]
That's true, but I'm not really happy with this project. The communication protocol is not great and many parts should be rewritten. It would also be much better if the majority of the code was shared with the desktop version. So I don't think it's worth publishing the app on F-Droid.

**Comment by cyanwolfg** [comment]
I see and understand better why it is not in F-droid, I think the application itself has a lot of potential but I also understand that it must be difficult to create an application from scratch.
